README.md

```diff
@@ -0,0 +1,713 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
+
+# Installation
+
+```console
+$ npm install diacritics-transliterator
+```
+
+# Usage
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.setVersion("v1");
```

Comment:
I'd vote to remove this from the default example, as we've discussed to make this optionally.